x-pack/plugins/ml/public/application/timeseriesexplorer/styles.ts

```diff
@@ -0,0 +1,331 @@
+        },
+      },
+
+      '.focus-chart': {
+        '.x-axis-background': {
+          line: {
+            fill: 'none',
+            shapeRendering: 'crispEdges',
+            stroke: euiThemeVars.euiColorLightestShade,
+          },
+          rect: {
+            fill: euiThemeVars.euiColorLightestShade,
+          },
+        },
+        '.focus-zoom': {
+          fill: euiThemeVars.euiColorDarkShade,
+          a: {
+            text: {
+              fill: euiThemeVars.euiColorPrimary,
+              cursor: 'pointer',
+            },
+            '&:hover, &:active, &:focus': {
+              textDecoration: 'underline',
+              fill: euiThemeVars.euiColorPrimary,
+            },
+          },
+        },
+      },
+
+      '.context-chart': {
+        '.x.axis path': {
+          display: 'none',
+        },
+        '.axis text': {
+          fontSize: '10px',
+          fill: euiThemeVars.euiTextColor,
+        },
+        '.values-line': {
+          strokeWidth: 1,
+        },
+        '.mask': {
+          polygon: {
+            fillOpacity: 0.1,
+          },
+          '.area.bounds': {
+            fill: euiThemeVars.euiColorLightShade,
+          },
+          '.values-line': {
+            strokeWidth: 1,
+            stroke: euiThemeVars.euiColorMediumShade,
+          },
+        },
+      },
+
+      '.swimlane .axis text': {
+        display: 'none',
+      },
+
+      '.swimlane rect.swimlane-cell-hidden': {
+        display: 'none',
+      },
+
+      '.brush .extent': {
+        fillOpacity: 0,
+        shapeRendering: 'crispEdges',
+        stroke: euiThemeVars.euiColorDarkShade,
+        strokeWidth: 2,
+        cursor: 'move',
+        '&:hover': {
+          opacity: 1,
+        },
+      },
+
+      '.top-border': {
+        fill: euiThemeVars.euiColorEmptyShade,
+      },
+
+      'foreignObject.brush-handle': {
+        pointerEvents: 'none',
+        paddingTop: '1px',
+      },
+
+      'div.brush-handle-inner': {
+        border: `1px solid ${euiThemeVars.euiColorDarkShade}`,
+        backgroundColor: euiThemeVars.euiColorLightShade,
+        height: '70px',
+        width: '10px',
+        textAlign: 'center',
+        cursor: 'ew-resize',
+        marginTop: '9px',
+        fontSize: euiThemeVars.euiFontSizeS,
+        fill: euiThemeVars.euiColorDarkShade,
+      },
+
+      'div.brush-handle-inner-left': {
+        borderRadius: `${euiThemeVars.euiBorderRadius} 0 0 ${euiThemeVars.euiBorderRadius}`,
+      },
+
+      'div.brush-handle-inner-right': {
+        borderRadius: `0 ${euiThemeVars.euiBorderRadius} ${euiThemeVars.euiBorderRadius} 0`,
+      },
+
+      'rect.brush-handle': {
+        strokeWidth: 1,
+        stroke: euiThemeVars.euiColorDarkShade,
+        fill: euiThemeVars.euiColorLightShade,
+        pointerEvents: 'none',
+        '&:hover': {
+          opacity: 1,
+        },
+      },
+    },
+
+    // Annotations styles
+
+    '.ml-annotation': {
+      '&__brush': {
```

Comment:
This `&__...` syntax here and down below unfortunately isn't working after the migration. SCSS will use `&` to produce class names like `.ml-annotation__brush`, that's no longer happening so the correct styles are not getting applied to annotations.

This is how it looks on `main`:

<img width="901" alt="image" src="https://github.com/user-attachments/assets/737df2fb-6d1b-4f70-81dd-42bc03cef971">

This PR:

<img width="896" alt="image" src="https://github.com/user-attachments/assets/48fd8498-2cee-4ddc-9d36-fcd8d96fab34">

Reply:
I believe the syntax is correct and it works!
What happened is (I'm guessing) that I deleted the `@import 'timeseriesexplorer/index';` in
 `x-pack/plugins/ml/public/application/_index.scss`
 So the styles for `annotations` in the `Anomaly Explorer` were removed.
 In [0d5af5f](https://github.com/elastic/kibana/pull/195259/commits/0d5af5f5e20ed4cb007068523929eb0079e7e25a) I split the styles into two separate functions and applied them where needed. It seems to be working correctly now. I've also updated the embeddables

Reply:
Oh nice, great to see that's supported, so I was just misinterpreting why it was broken. I checked the latest state and it's working for me now too 👍